shared/wallets/link-existing/index.stories.js

```diff
@@ -18,7 +18,6 @@ const common = {
   onDone: action('onDone'),
   onKeyChange: action('onKeyChange'),
   onNameChange: action('onNameChange'),
-  onViewChange: action('onViewChange'),
```

Comment:
`onViewChange` wasn't being called anywhere in the `LinkExisting` component; it wasn't even defined in its props.